Renewed/nl_nl.json

```diff
@@ -361,6 +361,88 @@
 	"block.lotr.nettles": "Brandnetels",
 	"block.lotr.fernsprout": "Varenspruit",
 	"block.lotr.candle": "Kaars",
+	"block.lotr.gold_chandelier": "Gouden kroonluchter",
+	"block.lotr.iron_chandelier": "Ijzeren kroonluchter",
```

Comment:
`Ij` mag volgens mij niet; moet `IJ` zijn

Reply:
Klopt, heb je gelijk in.